### DISCUSSION TITLE
Kerberos support

### DISCUSSION THREAD

**Discussion by smiralexan** [redis/lettuce#1925] [Q&A]
Lettuce does support SSL - https://github.com/lettuce-io/lettuce-core/wiki/SSL-Connections.
Our app needs support of Kerberos authentication. Is it possible to enable it using Lettuce currently?
I've seen that properties for Kerberos are passed through the Java SE standard library - https://docs.oracle.com/javase/8/docs/technotes/guides/security/jgss/tutorials/KerberosReq.html. 
If we set these properties correctly, can Lettuce use them under the hood?

**Comment by mp911de** [comment]
Redis doesn't support Kerberos.
